src/hooks/xcm/useTransferRouter.ts

```diff
@@ -0,0 +1,383 @@
+import { ethers } from 'ethers';
+import { SelectedToken } from 'src/c-bridge';
+import { endpointKey } from 'src/config/chainEndpoints';
+import { useAccount, useBalance, useNetworkInfo } from 'src/hooks';
+import {
+  astarChains,
+  Chain,
+  checkIsRelayChain,
+  removeEvmName,
+  XcmChain,
+  xcmChains,
+  xcmToken,
+} from 'src/modules/xcm';
+import { generateAssetFromEvmToken, generateNativeAsset } from 'src/modules/xcm/tokens';
+import { useStore } from 'src/store';
+import { Asset } from 'src/v2/models';
+import { computed, ref, watch, watchEffect } from 'vue';
+import { useRoute, useRouter } from 'vue-router';
+import { EvmAssets, XcmAssets } from './../../store/assets/state';
```

Comment:
tiny point : it's better to use not relative path by changing to `src/store/assets/state`

Reply:
I've changed almost all the importing styles to the absolute path instead of the relative path in [this](https://github.com/AstarNetwork/astar-apps/pull/479/commits/d356fb33de77d5ce2b34fcbc7406834544f22dfb) commit. But could you help define the rule for importing format? I think we should all be on the same page from now on. @bobo-k2 @hoonsubin @niklabh